### PR TITLE
218831 update headings for bacnet

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/configuration/configuration.md [hunk 1]
```diff
@@ -14,10 +14,18 @@ For more information on PI Adapter configuration tools, see [Configuration tools
...
+1. Configure one or several BACnet system components.
 
-2. Configure a BACnet data source for each BACnet device.<br>See [PI Adapter for BACnet data source configuration](xref:PIAdapterforBACnetDataSourceConfiguration#configure-bacnet-data-source).
+    See <xref:SystemComponentsConfiguration>.
 
-3. Configure a BACnet data selection for each BACnet data source.<br>See [PI Adapter for BACnet data selection configuration](xref:PIAdapterforBACnetDataSelectionConfiguration#configure-bacnet-data-selection).
+1. Configure a BACnet data source for each BACnet device.
 
-4. Configure one or several egress endpoints.<br>See [Egress endpoints configuration](xref:EgressEndpointsConfiguration).
+    See <xref:PIAdapterforBACnetDataSourceConfiguration>.
```
Comment (TinaUll) [inline content/configuration/configuration.md:23]: You might want to add the other configuration topics as well.

File: content/installation/install-pi-adapter-for-bacnet-using-docker.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
 uid: InstallPIAdapterForBACnetUsingDocker
 ---
 
-# Install PI Adapter for BACnet using Docker
+# Install using Docker
```
Comment (TinaUll) [inline content/installation/install-pi-adapter-for-bacnet-using-docker.md:5]: Same, I would use Installation

**Review by TinaUll — APPROVED** [review]

File: content/configuration/configuration.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/configuration/configuration.md:25]: Yeah, after I saw your feedback on AEH, I added them here as well.

File: content/installation/install-pi-adapter-for-bacnet-using-docker.md (hunk 2 above)
Comment (osisoft-mbishop) [reply content/installation/install-pi-adapter-for-bacnet-using-docker.md]: Updated.

**Pull request merged**